modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/edit_role_assignments.jsp

```diff
@@ -149,7 +149,14 @@ HashMap<String, Object> additionalProps = HashMapBuilder.<String, Object>put(
 	</c:choose>
 </aui:form>
 
-<liferay-frontend:component
-	context="<%= additionalProps %>"
-	module="js/EditRoleAssignments"
-/>
\ No newline at end of file
+<aui:script require='<%= npmResolvedPackageName + "/js/EditRoleAssignments as EditRoleAssignments" %>'>
+	EditRoleAssignments.default({
+		editRoleAssignmentsURL:
+			'<%= additionalProps.get("editRoleAssignmentsURL") %>',
+		modalSegmentState: '<%= additionalProps.get("modalSegmentState") %>',
+		namespace: '<portlet:namespace />',
+		portletURL: '<%= additionalProps.get("portletURL") %>',
+		roleName: '<%= additionalProps.get("roleName") %>',
+		selectAssigneesURL: '<%= additionalProps.get("selectAssigneesURL") %>',
+	});
```

Comment:
IMO if we are using an inline script, then there is no need for the EditRoleAssignments component to exist. We can just inline the logic and require the `add_assignees` script directly.

Same with this usage of `additionalProps`, we instead just can just pass in the values directly as defined on the JSP.

Reply:
Ok, yes that also makes sense.
I updated that in [https://github.com/liferay-frontend/liferay-portal/pull/835/commits/42f0e6ff589a4545beec59654d1276efa99730c6](https://github.com/liferay-frontend/liferay-portal/pull/835/commits/42f0e6ff589a4545beec59654d1276efa99730c6), let me know what you think when you can.